app/src/main/res/values/dimens.xml

```diff
@@ -596,6 +596,12 @@
   <!-- StoryFragment -->
   <dimen name="story_fragment_padding_bottom">160dp</dimen>
 
+<!--  RevisionCardFragment-->
+  <dimen name="revision_card_fragment_padding_bottom">160dp</dimen>
+  <dimen name="revision_card_fragment_layout_margin">28dp</dimen>
```

Comment:
If you just mentioned `margin` it means that its equivalent in all sides top,bottom,start,end but instead its equal only for start and end.
So instead prefer using
`<dimen name="revision_card_fragment_layout_margin_start">28dp</dimen>`
`<dimen name="revision_card_fragment_layout_margin_end">28dp</dimen>`


Ditto for all files.

Reply:
changed the naming